lib/routes/ln_address/ln_address_fee_message.dart

```diff
@@ -0,0 +1,119 @@
+import 'dart:math';
+
+import 'package:breez_sdk/bridge_generated.dart';
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:c_breez/bloc/account/account_bloc.dart';
+import 'package:c_breez/bloc/account/account_state.dart';
+import 'package:c_breez/bloc/currency/currency_bloc.dart';
+import 'package:c_breez/bloc/lsp/lsp_bloc.dart';
+import 'package:c_breez/bloc/payment_options/payment_options_bloc.dart';
+import 'package:c_breez/widgets/warning_box.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class LnAddressFeeMessage extends StatelessWidget {
+  const LnAddressFeeMessage();
+
+  @override
+  Widget build(BuildContext context) {
+    final themeData = Theme.of(context);
+    final lspState = context.watch<LSPBloc>().state;
+    final isChannelOpeningAvailable = lspState?.isChannelOpeningAvailable ?? false;
+    final openingFeeParams = lspState?.lspInfo?.openingFeeParamsList.values.first;
+
+    return isChannelOpeningAvailable
+        ? WarningBox(
+            boxPadding: const EdgeInsets.fromLTRB(16, 30, 16, 16),
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Text(
+                  _formatFeeMessage(context, openingFeeParams!),
+                  style: themeData.textTheme.titleLarge,
+                  textAlign: TextAlign.center,
+                ),
+              ],
+            ),
+          )
+        : const SizedBox();
+  }
+
+  String _formatFeeMessage(BuildContext context, OpeningFeeParams openingFeeParams) {
+    final texts = context.texts();
+    final currencyState = context.read<CurrencyBloc>().state;
+    final accountState = context.read<AccountBloc>().state;
+    final paymentOptionsState = context.read<PaymentOptionsBloc>().state;
+
+    final minFee = openingFeeParams.minMsat ~/ 1000;
+    final minFeeFormatted = currencyState.bitcoinCurrency.format(minFee);
+    final minFeeAboveZero = minFee > 0;
+    final setUpFee = (openingFeeParams.proportional / 10000).toString();
+    final liquidity = currencyState.bitcoinCurrency.format(
+      accountState.maxInboundLiquidity,
+    );
+    final liquidityAboveZero = accountState.maxInboundLiquidity > 0;
+
+    final channelFeeLimitSat = paymentOptionsState.channelFeeLimitMsat ~/ 1000;
+    final maxFee = maxReceivableSat(accountState, channelFeeLimitSat, openingFeeParams);
+    final maxFeeFormatted = currencyState.bitcoinCurrency.format(maxFee);
+    final isFeesApplicable = maxFee > accountState.maxInboundLiquidity;
+
+    if (!isFeesApplicable) {
+      // Send more than {minSats} and up to {maxSats} to this address. This address can be used only once.
```

Comment:
The LNURL can be reused
```suggestion
      // Send more than {minSats} and up to {maxSats} to this address.
```

Reply:
Ok, removing all the "This address can be used only once." texts.

Reply:
Done c73b1a7e89afb8b65fa0b6f6ff374e6f2cc6fbcc

Breez-Translations PR: https://github.com/breez/Breez-Translations/commit/94f49c9cbeb81890660f3e7ead1573dbf59ed697